gst/tensor_mux/gsttensormux.c

```diff
@@ -378,6 +456,10 @@ gst_tensor_mux_collect_buffer (GstTensorMux * tensor_mux,
 
       mem = gst_buffer_get_memory (buf, 0);
       gst_buffer_append_memory (tensors_buf, mem);
+      pad->buffer = buf;
```

Comment:
line 459 pad->buffer = buf;
is it necessary? if so, remove line 470 (gst_buffer_unref (buf);  this makes pad->buffer unstable)

Reply:
This appears critical.